ledger-tool/src/main.rs

```diff
@@ -818,8 +854,35 @@ fn open_blockstore(
         },
     ) {
         Ok(blockstore) => blockstore,
+        Err(BlockstoreError::RocksDb(err))
+            if (err
+                .to_string()
+                .starts_with("Invalid argument: Column family not found:")
+                || err
+                    .to_string()
+                    .starts_with("IO error: No such file or directory:"))
+                && access_type == AccessType::Secondary =>
+        {
+            info!("Blockstore is missing column(s) and unable to create with Secondary access");
```

Comment:
If we have included other rocksdb error like "IO error: No such file or directory:", then this error message needs an if-bracket.

Reply:
Good point - I changed the message to be more generic. Let me know what you think - we could be more explicit by having several `if` statements ... this only runs once so while I don't like bloating things, it's not a perf hit either